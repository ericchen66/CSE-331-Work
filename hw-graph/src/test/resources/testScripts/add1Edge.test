# Tests adding one edge between two nodes of a graph,
# ensuring the parent and child node are correct

# Creates a graph
CreateGraph graph1

# Adds two nodes to the graph
AddNode graph1 node1
AddNode graph1 node2

# Creates an edge between the two nodes
AddEdge graph1 node1 node2 edge1

# Lists the child of parent node
ListChildren graph1 node1