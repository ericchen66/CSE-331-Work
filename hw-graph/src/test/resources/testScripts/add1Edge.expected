# Tests adding one edge between two nodes of a graph,
# ensuring the parent and child node are correct

# Creates a graph
created graph graph1

# Adds two nodes to the graph
added node node1 to graph1
added node node2 to graph1

# Creates an edge between the two nodes
added edge edge1 from node1 to node2 in graph1

# Lists the child of parent node
the children of node1 in graph1 are: node2(edge1)